src/execution/as_abi.rs

```diff
@@ -31,18 +29,17 @@ pub(crate) fn assembly_script_transfer_coins(
     to_address: i32,
     raw_amount: i64,
 ) -> ABIResult<()> {
-
     let env = ctx.data().clone();
-    sub_remaining_gas(&env, &mut ctx, settings::metering_transfer())?;
+    sub_remaining_gas_abi(&env, &mut ctx, function_name!())?;
     if raw_amount.is_negative() {
         abi_bail!("Negative raw amount.");
     }
     let memory = get_memory!(env);
-    let to_address = &get_string(memory, &mut ctx, to_address)?;
-    if cfg!(feature = "gas_calibration") {
-        let fname = format!("massa.{}:0", function_name!());
-        param_size_update(&env, &mut ctx, &fname, to_address.len(), true);
-    }
+    let to_address = &read_string(memory, &mut ctx, to_address)?;
+    // if cfg!(feature = "gas_calibration") {
```

Comment:
Why is this commented?

Reply:
I commented all of the param size calculation as they take time and not used anymore. I let it in comment so that we keep it if we want to use it in a special feature in the futur.

Reply:
If we want to be precise it's better to not have any function call I think it's better to comment than making a call that does nothing after.

Reply:
Ok so maybe add a comment so a future dev won't remove it?

Reply:
Done.